main.tex

```diff
@@ -579,23 +581,25 @@ \section*{Results and discussion}\label{sec:ResultsAndDiscussion}
 Fig~\ref{fig:uml} and Table~\ref{tab:entitiesandactivities} provide overviews of these entity and activity types and should be consulted when skipping the first section.
 
 Second, we will discuss our findings applying our specialization of PROV-DM and demonstrate the relationships as well as specific features of the provenance information of the 19 \wnt{} simulation studies covered in this publication.
+% TODO comma?
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 \subsection*{Further steps towards a PROV-DM ontology for cellular biochemical simulation models}
 
 %Introduction to our specialization of PROV-DM
-The specialization of PROV-DM, which has been introduced by Ruscheinski et al. (2018)~\cite{Ruscheinski2018}, is being revised and refined.
+The specialization of PROV-DM, which was introduced by Ruscheinski et al. (2018)~\cite{Ruscheinski2018}, has been revised and refined.
 For capturing provenance information of simulation studies of cellular biochemical simulation models and relating these, we are defining and using
 \begin{inparaenum}[a)]
 \item specific types of entities and activities and
 \item specific relations with their roles and constraints.
 \end{inparaenum}
-We have identified these types and relations as well as information that is useful for describing them in a succinct manner during the process of collecting provenance information of the studies containing cellular biochemical models.
+During the process of collecting provenance information from the studies, we identified the types and relations as well as information that was useful for describing them.
 Our final set of entities, activities, and relations is shown in Table~\ref{tab:entitiesandactivities}.
-Each entity and activity has already been mentioned for provenance, modeling or documentation purposes, or experiment design of simulation studies~\cite{Ruscheinski2017, Balci2012, Yilmaz2016, Monks2018, Bergmann2014, Carusi2012, Erdemir2012, Lorig2017, Ruscheinski2018, Waltemath2011, Wilsdorf2020b}, but they have not been used in this concrete combination.
+Each entity and activity has already been mentioned for provenance, modeling or documentation purposes, or experiment design of simulation studies~\cite{Ruscheinski2017, Balci2012, Yilmaz2016, Monks2018, Bergmann2014, Carusi2012, Erdemir2012, Lorig2017, Ruscheinski2018, Waltemath2011, Wilsdorf2020b}, but they have not all been used together.
 
 In the following, we will describe these entities, activities, and relations and discuss the information that should be included in \webprov{}.
```

Comment:
"In the following" what exactly?

Reply:
part?!

Reply:
or section?

Reply:
section I think sounds better? both would be fine haha